infer/crates/e2k-py/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "e2k-py"
+version = "0.1.0"
+edition.workspace = true
+
```

事故の可能性は今のところ無いけど、明記した方がよさそう？

```suggestion
publish = false

```